Example/ViewController.swift

```diff
@@ -22,6 +23,39 @@ class ViewController: UITableViewController {
         }
     }
 
+    @IBAction func actionShowMiniAppById() {
+        let alert = UIAlertController(title: "Please enter Mini App ID",
+                                      message: nil,
+                                      preferredStyle: .alert)
+
+        alert.addTextField { (textField) in
+            textField.keyboardType = .asciiCapable
+        }
+
+        let okAction = UIAlertAction(title: "OK", style: .default) { (_) in
+            self.showProgressIndicator()
+            if let textField = alert.textFields?.first, let miniAppID = textField.text, miniAppID.count > 0 {
+                MiniApp.info(miniAppId: miniAppID) { (result) in
+                    switch result {
+                    case .success(let responseData):
+                        self.currentMiniAppInfo = responseData
+                        self.performSegue(withIdentifier: "DisplayMiniApp", sender: nil)
+                    case .failure(let error):
+                        print(error.localizedDescription)
+                        self.displayErrorAlert(title: "Error", message: "Couldn't retrieve Mini App info, please try again later", dismissController: false)
+                    }
+                    self.dismissProgressIndicator()
+                }
+            } else {
+                self.dismissProgressIndicator()
+                self.displayErrorAlert(title: "Error", message: "Incorrect Mini App ID, please try again", dismissController: false)
```

Comment:
As of now, this might not display an error alert message "Incorrect ..." because dimissProgressIndicator will dismiss displayErrorAlert immediately. I faced a weird issue.
You can remove `self.dismissProgressIndicator()` and then you can set the `dismissController: true` in `self.displayErrorAlert()`